packages/web_ui/src/components/InstanceList.jsx

```diff
@@ -1,12 +1,13 @@
 import React from "react";
 import { useHistory } from "react-router-dom";
-import { Table } from "antd";
+import { notification, Button, Table } from "antd";
+import { CopyOutlined } from "@ant-design/icons";
```

Comment:
Do no import from ant-design/icons, it's got 3000 sub-modules and breaks development tools. Use
```js
import CopyOutlined from "@ant-design/icons/CopyOutlined";
```